### ISSUE TITLE
NullReference Exception when SqlTypeName is missing in JSON

### ISSUE THREAD

**Issue by p-i-p** [nuetzliches/spocr#56]
```
Unhandled Exception: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at SpocR.Generator.ParseTypeFromSqlDbTypeName(String sqlTypeName, Boolean isNullable) in /home/travis/build/nuetzliches/spocr/src/Generator.cs:line 42
   at SpocR.Generator.GetModelTextForStoredProcedure(Schema schema, StoredProcedure storedProcedure) in /home/travis/build/nuetzliches/spocr/src/Generator.cs:line 91
   at SpocR.Generator.GenerateDataContextModels(Boolean isDryRun) in /home/travis/build/nuetzliches/spocr/src/Generator.cs:line 249
   at SpocR.Managers.SpocrManager.Build(Boolean isDryRun) in /home/travis/build/nuetzliches/spocr/src/Managers/SpocrManager.cs:line 295
   at SpocR.Commands.RebuildCommand.OnExecute() in /home/travis/build/nuetzliches/spocr/src/Commands/RebuildCommand.cs:line 22
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at McMaster.Extensions.CommandLineUtils.Conventions.ExecuteMethodConvention.Invoke(MethodInfo method, Object instance, Object[] arguments) in C:\projects\commandlineutils\src\CommandLineUtils\Conventions\ExecuteMethodConvention.cs:line 86
   at McMaster.Extensions.CommandLineUtils.Conventions.ExecuteMethodConvention.OnExecute(ConventionContext context) in C:\projects\commandlineutils\src\CommandLineUtils\Conventions\ExecuteMethodConvention.cs:line 64
   at McMaster.Extensions.CommandLineUtils.Conventions.ExecuteMethodConvention.<>c__DisplayClass0_0.<<Apply>b__0>d.MoveNext() in C:\projects\commandlineutils\src\CommandLineUtils\Conventions\ExecuteMethodConvention.cs:line 25
--- End of stack trace from previous location where exception was thrown ---
   at McMaster.Extensions.CommandLineUtils.CommandLineApplication.<>c__DisplayClass126_0.<OnExecute>b__0() in C:\projects\commandlineutils\src\CommandLineUtils\CommandLineApplication.cs:line 505
   at SpocR.Program.Main(String[] args) in /home/travis/build/nuetzliches/spocr/src/Program.cs:line 63
```

**Comment by fen89** [comment]
EDIT: the whole problem was due to using json, meaning the below stuff is not relevant anymore.

@p-i-p Can you provide a bit more information? The exception states that there must be a problem with the parsed sql db type name instead of the stored procedure name. Can you provide the part of the `spocr.json` for the stored procedure which fails, so that I can reproduce the issue - fixing is not the problem, but I would like to see exactly why this is happening and the string input param is null.

The error occurs if the param `sqlTypeName` is null (I modified the input param):
![grafik](https://user-images.githubusercontent.com/4036376/70840811-40e33500-1e15-11ea-98fc-449b2ff3541a.png)

I need to see these props - I can only reproduce the error above if I delete an `SQLTypeName` property:

![grafik](https://user-images.githubusercontent.com/4036376/70840873-da124b80-1e15-11ea-86da-b8a440ea2cb3.png)

**Comment by fen89** [comment]
should be fixed with 43d77e8
